polaris/ocean/tests/single_column/single_column.cfg

```diff
@@ -0,0 +1,66 @@
+# Options related to the vertical grid
+[vertical_grid]
+
+# the type of vertical grid
+grid_type = uniform
+
+# Number of vertical levels
+vert_levels = 100
+
+# Depth of the bottom of the ocean
+bottom_depth = 400.0
+
+# The type of vertical coordinate (e.g. z-level, z-star)
+coord_type = z-star
+
+# Whether to use "partial" or "full", or "None" to not alter the topography
+partial_cell_type = None
+
+# The minimum fraction of a layer for partial cells
+min_pc_fraction = 0.1
```

Comment:
These may be fine here or may need to go into `cvmix/cvmix.cfg`.

Reply:
@xylar Rather than move the `vertical_grid` and `single_column` options to `cvmix/cvmix.cfg` should we instead make use of the templating so that future test cases can use these options but change values more easily? I'm imagining that `initial_state` will be basically used as-written for most single_column tests (e.g., the BGC tests would just be adding more variables on top of a piecewise linear T,S profile).

Reply:
Let's just keep things like this for now, and we can cross that bridge when we come to it.

Reply:
Ok, thanks. To be honest, I'm not quite sure where this one will go. We'll see!